docs/src/pages/Head.tsx

```diff
@@ -26,21 +27,18 @@ export const Head = () => {
     throw new Error(`Meta Info missing on ${filepath}`);
   }
 
-  const pageTitle = `${metaTitle ?? title} | ${capitalizeString(
-    platform
-  )} - Amplify UI`;
+  const homepagePaths = ['/', '/[platform]'];
 
-  const homepagePaths = [
-    '/',
-    ...FRAMEWORKS.map((framework) => `/${framework}`),
-  ];
+  const pageTitle = homepagePaths.includes(pathname)
```

Comment:
Homepage title is special, we don't want to append ' - Amplify. UI' because 'Amplify UI' will be at the start.